src/bms/command/channel/mapper.rs

```diff
@@ -33,14 +32,15 @@ fn key_layout_beat_to_channel_id(beat: KeyLayoutBeat) -> NoteChannelId {
         _ => '1', // Default fallback
     };
 
-    NoteChannelId::try_from([first_char as u8, second_char as u8]).unwrap()
+    let Ok(channel_id) = NoteChannelId::try_from([first_char as u8, second_char as u8]) else {
+        panic!("generated note channel id should be valid");
+    };
+    channel_id
```

Comment:
I think those of panicking with message of invariant are suitable to use `expect` over `panic!`. I don't want to maintain about what format to print as possible.